testing/test_pickle_checkpoint_step.py

```diff
@@ -12,7 +12,7 @@
 expected_outputs = np.ones((2, 2))
 
 
-def test_should_save_checkpoint_pickle(tmpdir):
+def test_should_save_checkpoint_pickle(tmpdir: str):
```

Comment:
Correct me if I'm wrong, but I think that the correct type is a LocalPath as follow: 

```
from py._path.local import LocalPath

def test_should_save_checkpoint_pickle(tmpdir: LocalPath):
```

Reply:
right my bad